src/libaktualizr/primary/initializer.cc

```diff
@@ -180,23 +167,22 @@ InitRetCode Repository::initEcuRegister() {
     Json::Value primary_ecu;
     primary_ecu["hardware_identifier"] = ecu_serials[0].second;
     primary_ecu["ecu_serial"] = ecu_serials[0].first;
-    primary_ecu["clientKey"]["keytype"] = config.uptane.getKeyTypeString();
+    primary_ecu["clientKey"]["keytype"] = keyTypeToString(keys.getUptaneKeyType());
     primary_ecu["clientKey"]["keyval"]["public"] = primary_public;
     all_ecus["ecus"].append(primary_ecu);
   }
 
-  std::vector<std::shared_ptr<Uptane::SecondaryInterface> >::const_iterator it;
-  for (it = secondary_info.begin(); it != secondary_info.end(); it++) {
+  for (auto it = secondary_info.begin(); it != secondary_info.end(); it++) {
```

Comment:
Not actually a big deal, but is it possible to use cbegin() and cend() here?

Reply:
Yes, makes sense, thank you.